docs/api/topics/apps.rst

```diff
@@ -280,6 +281,8 @@ App
     :type release_notes: string|object|null
     :param resource_uri: The canonical URI for this resource.
     :type resource_uri: string
+    :param size: Size of the app in bytes.
+    :type size: int
```

Comment:
If there's more than one version, how does the API consumer know which one this belongs to? Would we ever want to show sizes for all versions?

If anything I'd say, `"Size of the app's current version in bytes."`.